<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteSheetSplitterWidget</class>
 <widget class="QWidget" name="SpriteSheetSplitterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="m_layout_2">
     <item>
      <widget class="GraphicsView" name="m_preview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>600</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::DragMode::RubberBandDrag</enum>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::ViewportAnchor::AnchorUnderMouse</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="ZoomWidget" name="m_zoom_widget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="m_sidebar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="m_layout">
        <item>
         <widget class="QPushButton" name="m_btn_open_texture">
          <property name="text">
           <string>Open Texture...</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/Resources.qrc">
            <normaloff>:/icons/image-x-generic</normaloff>:/icons/image-x-generic</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_btn_open_atlas">
          <property name="text">
           <string>Open Atlas..</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/Resources.qrc">
            <normaloff>:/icons/document-open</normaloff>:/icons/document-open</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="m_stacked_widget_properties">
        <widget class="QWidget" name="m_page_empty">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Please open a texture or an atlas</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="GridPropertyEditorWidget" name="m_page_grid"/>
        <widget class="AtlasPropertyEditorWidget" name="m_page_atlas"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="m_groupbox_export_sprites">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="m_label_sprite_format">
           <property name="text">
            <string>Sprite format</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="m_combo_sprite_format">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="m_btn_export_sprites">
           <property name="text">
            <string>Export sprites...</string>
           </property>
           <property name="icon">
            <iconset resource="../Resources/Resources.qrc">
             <normaloff>:/icons/document-save</normaloff>:/icons/document-save</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="m_layout_3">
        <item>
         <widget class="QPushButton" name="m_btn_save_atlas">
          <property name="text">
           <string>Save atlas...</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/Resources.qrc">
            <normaloff>:/icons/document-new</normaloff>:/icons/document-new</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="m_action_animate">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/icons/video-x-generic</normaloff>:/icons/video-x-generic</iconset>
   </property>
   <property name="text">
    <string>Animate</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomWidget</class>
   <extends>QWidget</extends>
   <header location="global">Sol2dTexturePackerGui/ZoomWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">Sol2dTexturePackerGui/GraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>GridPropertyEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">Sol2dTexturePackerGui/Splitter/GridPropertyEditorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AtlasPropertyEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">Sol2dTexturePackerGui/Splitter/AtlasPropertyEditorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_btn_open_texture</tabstop>
  <tabstop>m_btn_open_atlas</tabstop>
  <tabstop>m_combo_sprite_format</tabstop>
  <tabstop>m_btn_export_sprites</tabstop>
  <tabstop>m_btn_save_atlas</tabstop>
  <tabstop>m_preview</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
